### PR TITLE
Arush0311/mwi

### PR DESCRIPTION
**Pull request by arush0311**

### CODE REVIEW

File: app/scienceapi/urls.py [hunk 1]
```diff
@@ -14,14 +14,43 @@
...
     url(r'^study-groups/', include('scienceapi.study_groups.urls')),
     url(r'^resources/', include('scienceapi.resources.urls')),
+    url(r'^blog/', include('scienceapi.scienceblog.urls')),
 ]
+
+urlpatterns = [
+    url(r'^admin/', include(adminpatterns)),
+    url(r'^api/', include(apipatterns)),
+    url(r'^', include(apipatterns))
+]
```
Comment (arush0311) [inline app/scienceapi/urls.py]: including apipatterns to root too to avoid breaking frontend

**Comment by arush0311** [comment]
Introduced following env variables as of now:

**Comment by cadecairos** [comment]
@arush0311 So we need an S3 bucket, and credentials for staging and production?

File: app/scienceapi/scienceblog/views.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+from rest_framework.generics import ListAPIView, RetrieveAPIView
+
+from mezzanine.blog.models import BlogPost, BlogCategory
+
+from .serializers import BlogPostSerializer, BlogCategorySerializer
+
+
+def blog_preview(request, *args, **kwargs):
+    # Redirect to preview url on frontend. Using a dummy url for now.
+    return redirect('https://google.com')
```
Comment (alanmoo) [inline app/scienceapi/scienceblog/views.py]: Why do you have this?

File: app/scienceapi/templates/admin/base_site.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
+{% extends "admin/base.html" %}
+{% load mezzanine_tags i18n staticfiles %}
+
+{% block title %}{{ title }} | Mezzanine{% endblock %}
+
+{% block javascripts %}
+<script>
```
Comment (alanmoo) [inline app/scienceapi/templates/admin/base_site.html]: What's going on ehre?

Comment (arush0311) [reply app/scienceapi/templates/admin/base_site.html]: This is mezzanine. I have just overridden a couple of lines below. I wm using `override: start` tags for that

File: app/scienceapi/scienceblog/views.py (hunk 2 above)
Comment (arush0311) [reply app/scienceapi/scienceblog/views.py]: This is the link used inside preview button. We will change this once changes in frontend lands

**Comment by arush0311** [comment]
@cadecairos yes please!

**Comment by cadecairos** [comment]
@arush0311 unfortunately this isn't working because we didn't include `boto3` in requirements.txt

**Pull request merged**